app-emulation/virtualbox/files/virtualbox-7.0.6-gcc-13.patch

```diff
@@ -0,0 +1,28 @@
+Add a few missing includes.
+
+Bug: https://bugs.gentoo.org/892852
+
+--- a/src/libs/dxvk-native-1.9.2a/src/util/util_matrix.cpp
++++ b/src/libs/dxvk-native-1.9.2a/src/util/util_matrix.cpp
```

Comment:
I was going to say "send it upstream to dxvk-native" but https://github.com/Joshua-Ashton/dxvk-native is archived...

Is there a new location or is it gone?

Reply:
This is a copy of dxvk-native included in virtualbox sources.  I see they have quite a few, never noticed them before.  I should try to unbundle some of them